dev/integration_tests/flutter_gallery/android/app/build.gradle

```diff
@@ -74,4 +75,5 @@ dependencies {
     androidTestImplementation 'androidx.test:rules:1.1.1'
     androidTestImplementation 'androidx.test:runner:1.1.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
+    implementation 'androidx.multidex:multidex:2.0.1'
```

Comment:
@blasten `flutter_gallery` now goes over the 64K method limit and requires multidex.

Reply:
@GaryQian should this use the flag or is this dependency sufficient?

Reply:
`multiDexEnabled true` was addded to the `defaultConfig` section of this gradle file and `AndroidManifest.xml` now points to `androidx.multidex.MultiDexApplication` as the application class to use.

Reply:
This dependency is actually automatically added if `multiDexEnabled` is set to true. Adding this here is actually not necessary!

Reply:
Ok, removing this dependency. The docs for multidex instructs devs to add it which is why I did it   https://developer.android.com/studio/build/multidex#mdex-gradle

Reply:
Yeah, automatic multidex support is new. The manual way is also fine, but the built-in support is preferred to be more canonical.

Docs have not landed yet: https://github.com/flutter/website/pull/6400/files

Reply:
Ok, sorry for the confusion. Did not know flutter had this awesome new capability. Removed the manual configuration